docs/meetups/index.html

```diff
@@ -21,7 +21,7 @@
 -->
 
 <div class="meetups" data-plugin="paginate" data-max-posts="2" data-previous=".paginate-previous--meetup" data-next=".paginate-next--meetup" data-pagination=".pagination--meetup">
-    {% assign sorted = (site.meetups | sort: 'date') | reverse %}
+    {% assign sorted = site.meetups | sort: 'date' | reverse %}
```

Comment:
why do we need these changes?

Reply:
Yes this is due to the change from redcarpet to kramdown